backend.go

```diff
@@ -17,14 +17,14 @@ const (
 
 	// aliasNameSourceUnset provides backwards compatibility with preexisting roles.
 	aliasNameSourceUnset   = ""
-	aliasNameSourceSAToken = "sa_token"
-	aliasNameSourceSAPath  = "sa_path"
-	aliasNameSourceDefault = aliasNameSourceSAToken
+	aliasNameSourceSAUid   = "serviceaccount_uid"
+	aliasNameSourceSAPath  = "serviceaccount_path"
```

Comment:
@tvoran, @tomhjp  wasn't that keen on this parameter's name what do  you think? The Alias' name is computed from the SA token's `namespace` and `serviceaccount` delimited by `/`

Reply:
Maybe just `serviceaccount_name`? Since UID vs name seems to be the big difference.

Reply:
Thanks! Fixed in https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/112/commits/2fcb99aa83599cef93a5ef2ff6513004544e7c40